src/main/webapp/data/developers.json

```diff
@@ -380,6 +380,9 @@
       "name": "Dustin Lapierre",
       "username": "dustinlapierre"
     },
+    {
+      "username": "elaborateruse"
```

Comment:
I have 99% confidence that this person's name is Ed Kaplonski ([reference](https://github.com/TEAMMATES/teammates/pull/8458/commits/25d37c2cd5d4c69811d0b9a234aa417e2af91462))